auto_round/mllm/processor.py

```diff
@@ -42,6 +42,18 @@ def get_input(
             max_length=None,
             squeeze=True,
             **kwargs):
+        
+        if isinstance(text, list):
+            for message in text:
+                if not ("role" in message and "content" in message):
+                    raise ValueError(
+                        "When passing chat dicts as input, each dict must have a 'role' and 'content' key.")
+            continue_final_message = text[-1]["role"] == "assistant"
+
+            # add_generation_prompt=True will add <|im_start|>assistant to the end
+            text = tokenizer.apply_chat_template(
```

Comment:
The basic processor should not assume that the model is instruct-tuned; it may be using a base model instead

Reply:
is our default dataset only for instruct model?  Because I heard it contains multi turn chat text. If so, we should refine the code/data later to support base model